### ISSUE TITLE
Update from git servo/rust-geom to crates.io euclid

### ISSUE THREAD

**Issue by SimonSapin** [servo/servo#6406]
rust-geom has been renamed to euclid (because geom was already taken) and published on crates.io. We should update Servo and its dependencies accordingly. In `components/servo/Cargo.lock` I count 4 crates in other repositories that need to updated:

* [x] azure https://github.com/servo/rust-azure/
* [x] io-surface https://github.com/servo/rust-io-surface/
* [x] layers https://github.com/servo/rust-layers
* [x] offscreen_gl_context https://github.com/ecoal95/rust-offscreen-rendering-context

… and 16 in this repository:

* [ ] canvas
* [ ] canvas_traits
* [ ] compositing
* [ ] gfx
* [ ] glutin_app
* [ ] layout
* [ ] layout_traits
* [ ] msg
* [ ] net
* [ ] net_traits
* [ ] script
* [ ] script_traits
* [ ] style
* [ ] style_tests
* [ ] util
* [ ] util_tests

In each case:

* this section of `Cargo.toml`:
   ```toml
   [dependencies.geom]
   git = "https://github.com/servo/rust-geom"
   ```
   needs to be replaced with `geom = "0.1"` in the `[dependencies]` section

* `extern crate geom;` in `lib.rs` needs to be replaced with `extern crate euclid as geom;` or `extern crate euclid;`. In the latter case, the rest of the code also needs to be updated with the new name.

**Comment by Hockenberry** [comment]
Should this issue should be closed?

**Comment by SimonSapin** [comment]
Yes, thank. Fixed in #6423.
